benchmark/README.md

```diff
@@ -0,0 +1,19 @@
+# Benchmarks
+
+1) The following graphs show the benchmark results (runtime (in secs)) of running match attempts, using different PSI algorithms, on various Google Cloud VMs [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard) like n2-standard-32, n2-standard-48, n2-standard-64 and n2-standard-80. For each experiment, same type of VMs are used for both the sender and the receiver. The graphs are generated by performing match attempts between a receiver with a fixed dataset of 50m records and a sender with datasets of 50m, 100m, 150m, 200m and 250m records. The bpsi used for these experiments has a fixed false positive rate of 1e-6.
+
+![n2-standard-32](n2-standard-32.png)
+
+![n2-standard-48](n2-standard-48.png)
+
+![n2-standard-64](n2-standard-64.png)
+
+![n2-standard-80](n2-standard-80.png)
+
+2) Further, we provide a matrix of the running times of performing match attempts using different PSI algorithms on n2-standard-64 VMs between a receiver and a sender with datasets containing 50m, 100m, 200m, 300m, 400m and 500m records. The sizes of the receiver's datasets are represented row wise in each of the matrix, and the sizes of the sender's datasets are represented column wise in each of the matrix.
```

Comment:
The results for match attempts using different PSI algorithms are tabulated below. Both sender and receiver used n2-standard-64 VMS with datasets containing 50m, 100m, 200m, 300m, 400m and 500m records. The receiver's datasets are represented row-wise while the sender's datasets are represented column-wise.